canvas_modules/harness/cypress/integration/canvas/nodes.js

```diff
@@ -294,6 +294,30 @@ describe("Test from loaded file in legacy format", function() {
 	});
 });
 
+describe("Test new enableNodeLayout config parameter", function() {
+	beforeEach(() => {
+		cy.visit("/");
+		cy.openCanvasPalette("modelerPalette.json");
+	});
+
+	it("Test node 's height & width upon enableNodeLayout update", function() {
+		cy.clickToolbarPaletteOpen();
+		cy.clickCategory("Import");
+		cy.dragNodeToPosition("Database", 300, 200);
+
+		// Verify node height & width before updating enableNodeLayout
+		cy.verifyNodeTransform("Database", 8, 72);
```

Comment:
This is testing the position of the node which is unaffected by `enableNodeLayout`. What you need is a function that tests the height and width of the node rather than its position. If one doesn't currently exist you'll have to write one.